### DISCUSSION TITLE
Usage of ASGI option `asyncapi_path`

### DISCUSSION THREAD

**Discussion by yann-combarnous** [ag2ai/faststream#1977] [General]
This examples shows how to use ASGI to serve the documentation: https://faststream.airt.ai/latest/getting-started/asgi/?h=asyncapi_path#faststream-object-reuse.

However it does not mention how to pass asyncapi.json, and generates an empty documentation, any tip?

**Comment by Sehat1137** [comment]
@yann-combarnous We are going to support it in the next release (0.6.0)
